src/apresentacao/NotificacaoDeAusencia.java

```diff
@@ -37,7 +37,7 @@
     public NotificacaoDeAusencia() {
         initComponents();
         this.setLocationRelativeTo(null);
-        this.setIconImage(Toolkit.getDefaultToolkit().getImage("C:/Users/Thiago/Documents/NetBeansProjects/ProSub/mack_icon.jpg"));             
+        this.setIconImage(Toolkit.getDefaultToolkit().getImage(getClass().getResource("imagens\\logo_mackenzie.png")));             
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 132).](https://www.codacy.com/app/calebepb/substituicao/file/2759787669/issues/source?bid=3308597&fileBranchId=3349120#l40)